S21/myadlapalli/Language.py

```diff
@@ -0,0 +1,72 @@
+NO_CHANGE_LANGS = ['zh', 'ja', 'th']
+TURKISH_LANGS = ['tr', 'az']
+
+IRISH = 'ga'
+IRISH_VOWELS = ['A', 'E', 'I', 'O', 'U', 'Á', 'É', 'Í', 'Ó', 'Ú']
+
+GREEK = 'el'
+
+
+def toLower(sentence, lang):
+    sentence = sentence.strip()
```

Comment:
I think we went a bit out of scope, the problem was to read a word along with the language and then perform some lower operation to meet the rules described in the language rules.

Reply:
It would be better if we not hardcode the input.